backends/updates/macosx/macosx-updates.mm

```diff
@@ -74,11 +75,13 @@
 	// Finally give up our references to the objects
 	[menuItem release];
 
-	// Enable automatic update checking once a day (alternatively use
-	// checkForUpdates() here to check for updates on every startup)
-	// TODO: Should be removed when an update settings gui is implemented
-	setAutomaticallyChecksForUpdates(kUpdateStateEnabled);
-	setUpdateCheckInterval(kUpdateIntervalOneDay);
+	if (!ConfMan.hasKey("updates_check")
+			|| ConfMan.getInt("updates_check") == Common::UpdateManager::kUpdateIntervalNotSupported) {
+		setAutomaticallyChecksForUpdates(kUpdateStateDisabled);
+	} else {
+		setAutomaticallyChecksForUpdates(kUpdateStateEnabled);
+		setUpdateCheckInterval(ConfMan.getInt("updates_check"));
```

Comment:
I wonder if we should check that the getInt gives us something that is one of the value in the UpdateInterval  enum (or one of the value in the updateIntervals array, which is the same). If a user edits the file he can change it to anything, and I am not sure what Sparkle does when we call setUpdateCheckInterval with a negative or null NSTimeInterval. Also checking for updates every seconds or 10 seconds would not be a very good idea. Is there any case where it would be useful for the user to specify an interval in the config file that is not one of the predefined ones?
Note: if we add it the check should maybe be in setUpdateCheckInterval and not here.